sovrin_client/cli/helper.py

```diff
@@ -29,7 +29,7 @@ def getNewClientGrams():
         ADD_GENESIS_FORMATTED_REG_EX,
         # Regex for `new client steward with identifier <nym>`
         CLIENT_GRAMS_CLIENT_WITH_IDENTIFIER_FORMATTED_REG_EX,
-        # Regex for `client steward add sponsor bob` or `client steward
+        # Regex for `client steward add trustAnchor bob` or `client steward
```

Comment:
The correct role is TRUST_ANCHOR (I fixed the comment).
It's case-sensitive, as well as all other roles and txn names in CLI (SPONSOR was also case-sensitive).
I think we need an additional task/bug to make everything in CLI not case-sensitive (if it's needed).